wp-content/themes/mlibrary-twentyseventeen/style.css

```diff
@@ -119,9 +121,11 @@ h1, h2, h3, h4, h5, h6 {
   color: white;
 }
 
-/**
- * Additional styling for the gist embed.
- */
-.gist {
-  margin-bottom: .25em;
```

Comment:
This looks unrelated to the footer, was it intentional?

Reply:
This was not intentional. I meant to ask you about if we were using the embeds anymore and then was distracted and must have deleted the selection. I can add it back for now.